app/src/main/AndroidManifest.xml

```diff
@@ -242,6 +242,10 @@
             android:exported="false"
             android:label="@string/appName" />
 
+        <activity
+            android:name="com.duckduckgo.remote.messaging.impl.ui.ModalSurfaceActivity"
+            android:exported="false" />
+
```

Comment:
This is the new activity that will show the remote message on a "MODAL" surface (full screen).